### PR TITLE
Updates master-files styling

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Pretty sure this is everything.

Locates single letter vars: `/[A-Za-z]([, )]|$)`
Unspaced commas: `,([^ \\])` -> `, $1`
Unspaced equals signs: `([^ +/*-=])(=)([^ =])` -> `$1 $2 $3`
Unspaced other signs: `([^ +/*-=])(\+|-|\*)([^ =+-])`

Also removes a bit of commented out code that I came across, but I didn't run any searches for more. 

## Changelog

No player-facing changes

### CODE REVIEW

File: modular_skyrat/master_files/code/datums/traits/negative.dm [hunk 1]
```diff
@@ -70,8 +70,8 @@
 	name = "Mute"
 	desc = "Due to some accident, medical condition, or simply by choice, you are completely unable to speak."
 	value = -2 //HALP MAINTS
-	gain_text = "<span class='danger'>You find yourself unable to speak!</span>"
-	lose_text = "<span class='notice'>You feel a growing strength in your vocal chords.</span>"
+	gain_text = "<span class = 'danger'>You find yourself unable to speak!</span>"
+	lose_text = "<span class = 'notice'>You feel a growing strength in your vocal chords.</span>"
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/datums/traits/negative.dm]: Isn't there a way to make this use span macros?

File: modular_skyrat/master_files/code/modules/reagents/withdrawal/generic_addictions.dm [hunk 2]
```diff
@@ -7,17 +7,17 @@
...
+	switch(rand(1, 10))
 		if(1 to 3)
 			if(!HAS_TRAIT(affected_carbon, TRAIT_NOHUNGER))
-				affected_carbon.adjust_nutrition(-delta_time*10*strength)
+				affected_carbon.adjust_nutrition(-delta_time * 10 * strength)
 		if(3 to 5)
 			affected_carbon.emote("cough")
 		if(5 to 7)
 			to_chat(affected_carbon, span_warning("Your head hurts."))
 			affected_carbon.adjustStaminaLoss(4*strength)
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/reagents/withdrawal/generic_addictions.dm]: You missed one here.
```suggestion
			affected_carbon.adjustStaminaLoss(4 * strength)
```

File: modular_skyrat/master_files/code/modules/reagents/withdrawal/generic_addictions.dm [hunk 3]
```diff
@@ -7,17 +7,17 @@
...
 		if(8)
-			if(strength>=2)
+			if(strength >= 2)
 				to_chat(affected_carbon, span_warning("You feel a little dizzy."))
 				affected_carbon.Dizzy(3*strength)
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/reagents/withdrawal/generic_addictions.dm]: Here too.
```suggestion
				affected_carbon.Dizzy(3 * strength)
```

Comment (GoldenAlpharex) [reply modular_skyrat/master_files/code/modules/reagents/withdrawal/generic_addictions.dm]: You also missed one on line 25.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Good change.

File: modular_skyrat/master_files/code/datums/traits/negative.dm (hunk 1 above)
Comment (tf-4) [reply modular_skyrat/master_files/code/datums/traits/negative.dm]: Not sure tbh, I'd have to look into it.

Comment (tf-4) [reply modular_skyrat/master_files/code/datums/traits/negative.dm]: Looked into it and it works fine idk why it's not done anywhere on /tg/

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**